R/verb-arrange.R

```diff
@@ -1,3 +1,28 @@
+#' Arrange rows by variables in a local lazy tibble
+#'
+#' Order tbl_lazy rows by an expression involving its variables.
```

Comment:
This needs to mention `desc()`, like `?arrange()` does

Reply:
And shouldn't mention `tbl_lazy` since that won't mean anything to most people

Reply:
I've added a [note about `desc()`](https://github.com/tidyverse/dbplyr/pull/222/commits/6873d6482a1cfa076b4d49d0787485ab996cc3cd#diff-12fbbf8bf4196bb24a276dff57ec3116R7) and [removed `tbl_lazy`](https://github.com/tidyverse/dbplyr/pull/222/commits/6873d6482a1cfa076b4d49d0787485ab996cc3cd#diff-12fbbf8bf4196bb24a276dff57ec3116R6)